src/settings_internal.js

```diff
@@ -195,3 +195,7 @@ var STRUCT_INFO = '';
 
 // If true, building against Emscripten's asm.js/wasm heap memory profiler.
 var MEMORYPROFILER = 0;
+
+// Set to 1 by default for certain cases such as STANDALONE_WASM and
+// MINIMAL_RUNTIME
```

Comment:
This comment doesn't make sense. How about:

> By default (1) will import memory into WASM; certain modes such as STANDALONE_WASM and MINIMAL_RUNTIME will set to 0 and export memory from WASM instead.

(Though see my other comment about changing the name and value of this setting.)

Reply:
Done